backend/src/test/kotlin/blue/mild/covid/vaxx/util/PersonalNumberGenerator.kt

```diff
@@ -0,0 +1,26 @@
+package blue.mild.covid.vaxx.util
+
+/**
+ * Generates valid personal number.
+ */
+fun generatePersonalNumber(): String {
```

Comment:
No a presne takovyhle veci bych dal do toho, co udelal Honza. Protoze tam mi to prislo jako divnej spagetti code :(

Reply:
to je copy paste od neho, az se to zamerguje, tak to sjednotime, proto jsem to vyhodil takhle ven